docs/advanced/custom-map-controls.md

```diff
@@ -2,41 +2,23 @@
 
 ## Overriding The Default Map Controller
 
-To change the default behavior of map interaction, you can implement/extend the `MapControls`
-class add pass an instance to the `mapControls` prop of `InteractiveMap`.
+To change the default behavior of map interaction, you can provide a custom map control to the `mapControls` prop of `InteractiveMap`.
 
-A simple example to disable mouse wheel:
-```js
-  /// my-map-controls.js
-  import {experimental} from 'react-map-gl';
-
-  export default class MyMapControls extends experimental.MapControls {
-
-    constructor() {
-      super();
-      // subscribe to additional events
-      this.events = ['click'];
-    }
+This custom map control must offer the following interface:
+- `setOptions(options)` - called by `InteractiveMap` when props change.
 
-    // Override the default handler in MapControls
-    handleEvent(event) {
-      if (event.type === 'click') {
-        console.log('hi!');
-      }
-      return super.handleEvent(event);
-    }
-  }
-```
-Then pass it to the map during render:
 ```jsx
   const mapControls = new MyMapControls();
-  <ReactMapGL mapControls={mapControls} ... />
+
+  render() {
+    return <ReactMapGL mapControls={mapControls} ... />;
+  }
 ```
 
 
-## MapControls Interface
+## MapControls Class
```

Comment:
Should this be a separate entry in API Reference, rather than inside an article?

Could be marked by (Advanced) or (Experimental) like some deck.gl classes.